### PR TITLE
Add a first year announcement

### PR DESCRIPTION
**Pull request by certik**
Should we do an anniversary post?

### CODE REVIEW

File: _posts/2021-04-20-First-Year.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "First year of the Fortran website"
+category: newsletter
```
Comment (certik) [inline _posts/2021-04-20-First-Year.md:4]: We have to change the category...

**Comment by certik** [comment]
I was thinking a smaller post, thanking everybody and encouraging people to join, and giving pointers how to join, and include a short but up-to-date summary of our goals.

**Review by everythingfunctional — APPROVED** [review]
Looks good to me. Do we have an "announcements" category? Should we add one?

File: _posts/2021-04-20-First-Year.md (hunk 1 above)
Comment (milancurcic) [reply _posts/2021-04-20-First-Year.md:4]: Can it be just "post"?

File: _posts/2021-04-20-First-Year.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+author: Ondřej Čertík, ...
+---
+
+In April 2020 we created a website for the Fortran language at
+[fortran-lang.org](https://fortran-lang.org/). In exactly one year, it has
+gotten to be the first result when you search "Fortran" in Bing, Yahoo,
```
Comment (milancurcic) [inline _posts/2021-04-20-First-Year.md:10]:
```suggestion
[fortran-lang.org](https://fortran-lang.org/). In exactly one year, it grew
￼to be the first result when you search "Fortran" in Bing, Yahoo,
```

File: _posts/2021-04-20-First-Year.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+DuckDuckGo, Ecosia, Qwant, SearchEncrypt and the second result in Google (after
+the Wikipedia page for Fortran).
+
+The goal of the Fortran website is to maintain a neutral website where any
+Fortran user (expert or novice), compiler vendor (open source or commercial),
+Fortran Standards Committee member, enthusiast, supporter or anybody else
+interested is welcome to participate. The Fortran language has been invented in
+1956, and we are trying to be the stewards of the language and we welcome you
+to join us.
```
Comment (milancurcic) [inline _posts/2021-04-20-First-Year.md:19]:
```suggestion
The goal of the website is to maintain a neutral place where any
￼Fortran user (expert or novice), compiler vendor (open source or commercial),
￼Fortran Standards Committee member, enthusiast, supporter or anybody else
￼interested is welcome to participate. Fortran was invented in
￼1956, and we aim to be the stewards of the language and we welcome you
￼to join us.
```

**Review by milancurcic — APPROVED** [review]
It's great, thanks. I put in a few suggestions to reduce repetition and fix a tense.

**Review by LKedward — APPROVED** [review]
Thanks, this looks good to me!

File: _posts/2021-04-20-First-Year.md (hunk 1 above)
Comment (LKedward) [reply _posts/2021-04-20-First-Year.md:4]: The 'category' isn't displayed anywhere on site and it looks like the site is currently hard-coded to only display posts in the 'newsletter' category, which needs fixing.

https://github.com/fortran-lang/fortran-lang.org/blob/a80ff0e335b18500e775cd0fd62cbbded78a9dfd/news/index.html#L12

Comment (certik) [reply _posts/2021-04-20-First-Year.md:4]: I see. I changed it to "post", but that would not show up, so I changed it back to "newsletter".

**Comment by certik** [comment]
#build_preview

**Comment by certik** [comment]
Everything seems to work, I updated things based on the comments, thank you everybody!

I will merge it tomorrow morning and announce on Twitter. What should be the Twitter text?

```
First year of the Fortran website
https://fortran-lang.org/newsletter/2021/04/20/First-Year/
```

**Comment by everythingfunctional** [comment]
> Everything seems to work, I updated things based on the comments, thank you everybody!
> 
> I will merge it tomorrow morning and announce on Twitter. What should be the Twitter text?
> 
> ```
> First year of the Fortran website
> https://fortran-lang.org/newsletter/2021/04/20/First-Year/
> ```

The new Fortran website celebrates one year: https://fortran-lang.org/newsletter/2021/04/20/First-Year/

or

fortran-lang.org celebrates it's first year: https://fortran-lang.org/newsletter/2021/04/20/First-Year/

Just my 2 cents.

**Comment by milancurcic** [comment]
I like Brad's second one, but written like this (no apostrophe)

> fortran-lang.org celebrates its first year: https://fortran-lang.org/newsletter/2021/04/20/First-Year/

**Comment by certik** [comment]
Thanks Brad and Milan, yes, I'll post [your](https://github.com/fortran-lang/fortran-lang.org/pull/244#issuecomment-822808757) version.

**Comment by certik** [comment]
I am going ahead and merging.

**Comment by certik** [comment]
Here is the Tweet, I copied & pasted from Milan's post: https://twitter.com/fortranlang/status/1384527637737971718

**Comment by awvwgk** [comment]
#delete_preview

**Pull request merged**